doc/setting_up_sidewalk_environment/setting_up_sidewalk_product.rst

```diff
@@ -19,20 +19,34 @@ The tools required for provisioning are located in the repository (`sdk-nrf`_ an
 
    .. note::
       The default address of the the :file:`mfg.hex` file provided in the official Amazon Sidewalk documentation is incompatible with the nRF Connect SDK applications.
-      You must add the ``--addr 0xFF000`` argument to the :file:`provision.py` script in order to generate the :file:`mfg.hex` file that is compatible with the nRF Connect SDK memory map.
+      You must add the:
+
+      * ``--addr 0x17c000`` argument for `nrf54l15pdk_nrf54l15`_
+      * ``--addr 0xFF000`` argument for other boards
+
+      to the :file:`provision.py` script in order to generate the :file:`mfg.hex` file that is compatible with the nRF Connect SDK memory map.
 
    * If you are using the combined device JSON file obtained from the AWS IoT console, use the ``certificate_json`` parameter.
      It will specify this file as an input when running the provisioning script.
 
       .. code-block:: python
 
+         # for nRF54l15
+         python3 provision.py nordic aws --output_bin mfg.bin --certificate_json certificate.json --addr 0x17c000
+         # for other boards
```

Comment:
Maybe instead of repeating instructions we could mention it just once and mark it as a variable (?) (the directive will need to be changed):

 ```suggestion
      .. parsed-literal::
         :class: highlight

         python3 provision.py nordic aws --output_bin mfg.bin --certificate_json certificate.json --addr *argument*
```